dashboard/templates/admin/prototype/export.html

```diff
@@ -1,13 +1,65 @@
 {% extends "admin/base.html" %}
 
+{% load staticfiles %}
+{% block extrastyle %}
+  <link rel="stylesheet" type="text/css" href="{% static "admin/dashboard/styles/gov-uk-elements.css" %}"/>
+{% endblock %}
+
 {% block content %}
-<div class="pad">
-  <h1>Export</h1>
-
-  <form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-  <input class="button" type="submit" value="Export" />
-  </form>
+<div class="pad export">
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <h1 class="heading-medium util_mt-0">Export</h1>
+
+      <form method="post">
+        {% csrf_token %}
+
+        <div class="form-group">
+          <label class="form-label" for="payroll_date">
+            Enter month:
+            <span class="form-hint">For example 01 2016</span>
+          </label>
+          <input autocomplete="off" class="form-control" id="id_date_month" maxlength="2" name="date_month" placeholder="MM" size="3" type="text">
+          <input autocomplete="off" class="form-control" id="id_date_year" maxlength="4" name="date_year" placeholder="YYYY" size="5" type="text">
+        </div>
+
+        <div class="form-group">
+          <label class="form-label" for="id_project">Project:</label>
+          <select class="form-control" id="id_project" name="project">
+            <option value="" selected="selected">---------</option>
+            {% for value, display in form.fields.project.choices %}
```

Comment:
nice change. still a little problem. `---------` is appearing twice. also projects might be sorted by name?